metadata-ws/Jenkinsfile

```diff
@@ -14,6 +14,8 @@ pipeline {
     stagingHost = credentials('STAGINGHOST')
     fallbackHost = credentials('FALLBACKHOST')
     productionHost = credentials('PRODUCTIONHOST')
+    userInfoUri = credentials('USERINFOURI')
```

Comment:
userInfoUri is not used tokenInfoUri is used

Reply:
I observed jwt-key is enough, it has all required information. Neither user-info nor token-info keys are required. Will remove user-info key

Reply:
Since this property has been removed, marking the conversation as resolved